web/src/components/pages/contribution/speak/speak.tsx

```diff
@@ -229,26 +229,31 @@ class SpeakPage extends React.Component<Props, State> {
     return this.state.recordingStatus === 'recording';
   }
 
-  private handleKeyUprerecording = async (event: any) => {
-    let index = null;
+  private handleKeyUprerecording = async (event: KeyboardEvent) => {
+    let reRecordIndex = null;
     //for both sets of number keys on a keyboard with shift key
     if (event.code === 'Digit1' || event.code === 'Numpad1') {
-      index = 0;
+      reRecordIndex = 0;
     } else if (event.code === 'Digit2' || event.code === 'Numpad2') {
-      index = 1;
+      reRecordIndex = 1;
     } else if (event.code === 'Digit3' || event.code === 'Numpad3') {
-      index = 2;
+      reRecordIndex = 2;
     } else if (event.code === 'Digit4' || event.code === 'Numpad4') {
-      index = 3;
+      reRecordIndex = 3;
     } else if (event.code === 'Digit5' || event.code === 'Numpad5') {
-      index = 4;
+      reRecordIndex = 4;
+    } else if (event.key === "Backspace" || event.key === "x") {
```

Comment:
I used `key` instead of `code` since I think we should respect the user's choice of keyboard layout? I might miss some rationale for using `code` though.

Reply:
Agreed! Would you be interested in submitting a separate PR to fix the rest of these as well?

Reply:
Its probably easier if you just do it :)